app-backend/api-it/src/test/resources/endpoint-scopes.csv

```diff
@@ -229,6 +229,18 @@ Path,Domain:Action,Verb
 /api/campaigns/{campaignId}/permissions,campaigns:share,put
 /api/campaigns/{campaignId}/permissions,campaigns:share,post
 /api/campaigns/{campaignId}/permissions,campaigns:share,delete
+/api/annotation-projects/{campaignID}/label-class-groups,annotationProjects:read,get
+/api/annotation-projects/{campaignID}/label-class-groups,annotationProjects:update,post
+/api/annotation-projects/{campaignID}/label-class-groups/{groupID},annotationProjects:read,get
+/api/annotation-projects/{campaignID}/label-class-groups/{groupID},annotationProjects:update,put
+/api/annotation-projects/{campaignID}/label-class-groups/{groupID}/activate,annotationProjects:update,post
+/api/annotation-projects/{campaignID}/label-class-groups/{groupID}/deactivate,annotationProjects:update,delete
+/api/annotation-projects/{campaignID}/label-class-groups/{groupId}/label-classes,annotationProjects:read,get
+/api/annotation-projects/{campaignID}/label-class-groups/{groupId}/label-classes,annotationProjects:update,post
+/api/annotation-projects/{campaignID}/label-class-groups/{groupId}/label-classes/{labelClassID},annotationProjects:read,get
+/api/annotation-projects/{campaignID}/label-class-groups/{groupId}/label-classes/{labelClassID},annotationProjects:update,put
+/api/annotation-projects/{campaignID}/label-class-groups/{groupId}/label-classes/{labelClassID}/activate,annotationProjects:update,post
+/api/annotation-projects/{campaignID}/label-class-groups/{groupId}/label-classes/{labelClassID}/deactivate,annotationProjects:update,delete
```

Comment:
... these are all wrong, I forgot to change the route components, so it's just re-checking the existing annotation project tests. Will fix after RRP